.gitignore

```diff
@@ -52,4 +52,6 @@ storybook-static
 tsconfig.tsbuildinfo
 
 # to save thunder https://marketplace.visualstudio.com/items?itemName=rangav.vscode-thunder-client files
-.thunder
\ No newline at end of file
+.thunder
+
```

Comment:
unnecesary newline

Reply:
I moved `.idea` to `# misc` in this [commit](https://github.com/webstudio-is/webstudio-builder/pull/1579/commits/5b8c0fe01f8ae332c5b6bc01bd91b5241eb08dc4).